<ui version="4.0" >
 <class>Calculation</class>
 <widget class="QDialog" name="Calculation" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>363</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>363</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Qsalat - calculation options</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title" >
    <string>Calculation methods</string>
   </property>
   <widget class="QComboBox" name="calcBox" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable" >
     <bool>false</bool>
    </property>
    <property name="currentIndex" >
     <number>-1</number>
    </property>
    <property name="maxVisibleItems" >
     <number>7</number>
    </property>
    <property name="maxCount" >
     <number>7</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>10</y>
     <width>161</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title" >
    <string>Asr options</string>
   </property>
   <widget class="QRadioButton" name="shafiiButton" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Shafii</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="hanafiButton" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>30</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Hanafi</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>100</y>
     <width>171</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title" >
    <string>Duhr added minutes</string>
   </property>
   <widget class="QSpinBox" name="duhrBox" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>30</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>60</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>100</y>
     <width>161</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title" >
    <string>Higher Latitudes option</string>
   </property>
   <widget class="QCheckBox" name="highLatBox" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>30</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Higher Latitudes</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelButton" >
   <property name="geometry" >
    <rect>
     <x>280</x>
     <y>220</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>220</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Save</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>190</y>
     <width>171</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title" >
    <string>Hijri date adjustment</string>
   </property>
   <widget class="QSpinBox" name="hijriBox" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>30</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimum" >
     <number>-2</number>
    </property>
    <property name="maximum" >
     <number>2</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
